06-building-a-program-start-to-finish/reflecting-on-the-learning-experience/Output/README.md

```diff
@@ -0,0 +1,7 @@
+#### Output
+- **Using**: Medium
+- **Deliver**: a post reflecting on your learning experience while producing these outputs
+- **With**: *independently*
+
+#### Targeted Learning Outcomes
+- [Medium](https://medium.com/@henry.chukwu/from-bootcamp-to-checkpoint-849b651bc7bf#.n8zao9ugv)
```

Comment:
Good read! Moving on, you'd want to define some of the terms such as OKBBs, Simulations, Checkpoint etc... for the sake of an audience not within the Andela community.